base/exports.jl

```diff
@@ -577,6 +577,7 @@ export
     rot180,
     rotl90,
     rotr90,
+    @safeindices,
```

Comment:
Will add the docstring. The test happens as a consequence of the new `@test_throws` in `offsetarray.jl` and the fact that the remaining tests work.